### PR TITLE
Add :setcookie, grabbing cookie from file

### PR DESCRIPTION
**Pull request by Matthew-D12**
Hello, I'm that guy you were talking to yesterday, I did some other changes to my cookie from file code, so if the system can't get the AoC cookie automatically with `os.Getenv("AOC_TOKEN")` when running `:setday`, it grabs from a file `cookie.txt` instead.
When using `:setday` and the cookie file doesn't exist, it creates an empty file to prevent reading errors (there may be a better way to do this though)
To use `:setcookie` the user needs to get the AoC cookie using the guide https://github.com/wimglenn/advent-of-code-wim/issues/1 and type `:setcookie <cookievalue>`, the <cookievalue> being everything between `session=` and the first `;` on that string:

`:setcookie 53616c2457214f5fd61f09acd05afad5e48ace27ae4917cfd0863c706cd508be236b52e3d2f4e2f90a252cb`
(not a real cookie, just an example)

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -143,6 +143,16 @@ func main() {
 				eval = evaluator.New()
 				prev.eval = eval
 
+			case ":setcookie":
```
Comment (lukechampine) [inline main.go]: not sure if a command is really necessary here. Telling the user to put their token in `AOC_TOKEN` or `token.txt` seems good enough.

Maybe later we can try to automatically load the cookie from the browser, like some other AoC helpers do.

Comment (Matthew-D12) [reply main.go]: No problem, let me remove it then, just a moment

**Comment by Matthew-D12** [comment]
Removed `:setcookie`, user has the option to add token to `AOC_SESSION` or `cookie.txt` (please feel free to rename it as you see fit)

**Comment by Matthew-D12** [comment]
Also removed `createCookieFile()`, as now it will only be used from a single place

**Comment by lukechampine** [comment]
thanks!

**Comment by lukechampine** [comment]
(I renamed cookie -> token)

**Pull request merged**